src/common/constants/index.ts

```diff
@@ -0,0 +1 @@
+export * from "./colors";
```

Comment:
Either expose all modules under `src/common/constants` from here to be consistent or remove this file